submodules/LIB/hardware/syn/Makefile

```diff
@@ -9,6 +9,9 @@ include ../../config_build.mk
 
 REMOTE_BUILD_DIR=$(USER)/$(BUILD_DIR_NAME)
 REMOTE_SYN_DIR=$(REMOTE_BUILD_DIR)/hardware/syn
+PDK_ROOT := $(CURDIR)
+export PDK_ROOT
```

Comment:
this need to be added, the reason is because the pdk is installed where PDK_ROOT points, since the location of instalation want to be know, the this need to be exported

Reply:
who or what installs the PDK? Is it nix?

If somebody download iob.-soc will the pdk be installed?